src/feedinlib/open_FRED.py

```diff
@@ -147,7 +157,7 @@ def __init__(
         }[variables if variables in ["pvlib", "windpowerlib"] else None]
 
         self.locations = (
-            {(l.x, l.y): self.location(l) for l in locations}
+            {(lo.x, lo.y): self.location(lo) for lo in locations}
```

Comment:
This is because the style checker complains about `l`maybe being confused with `1`, right? Since this is a non issue for any reasonable programming font, I'd vote for ignoring this error. Otherwise I'd vote for `p` or `point` since `lo` sounds like an abbreviation of `low` and makes this line rather hard to read for me.
But this is more a note to myself to fix this later, than something stopping this review.